JST-683: Golem inspect

SewerynKras opened the pull request:
Depends on https://github.com/golemfactory/golem-js/pull/789

If you want to try it out before the other PR is merged, checkout the branch `feature/JST-683/golem-sdk-inspect` in `golem-js` and `npm link` it to this repo

mgordel commented on src/inspect/inspect.action.ts:
```diff
@@ -0,0 +1,91 @@
+  try {
+    await yagna.connect();
+  } catch (err) {
+    console.log(chalk.red(`Cannot connect to Yagna, check the app-key and URL`));
```
maybe a reason could be added here. Now `yagna.connect` throws `GolemPlatformError` so maybe it's worth adding a message at the end with the original reason...?

mgordel commented on src/inspect/inspect.action.ts:
```diff
@@ -0,0 +1,91 @@
+}
+
+function printSelectedColumns(obj: object, columns: string[]) {
+  if (columns[0] === "*") {
```
For me (macos) I have to use `--columns "*"`. Using `--column *` returns `{}`. Maybe it's worth to validate it.. ?

SewerynKras replied:
This is a feature of the shell you're using, if you don't wrap * in quotation marks it gets replaced with all files in the current dir (try `echo *`), there is nothing we can do about that from the cli perspective :/

mgordel reviewed (approved):
I tested locally and it works fine

mgordel commented:
Additionally, you also need to change `run-on-golem/shell-context.ts`, `await this.executor.end();` -> `await this.executor.terminate();` This was removed in the new beta release.

SewerynKras commented:
@mgordel thanks for the review, I addressed your comments, can you also take a look at the other part of this PR in golem-js: https://github.com/golemfactory/golem-js/pull/789 ?
